src/Windows/R/Editor/Test/Roxygen/RoxygenCompletionTest.cs

```diff
@@ -33,28 +34,37 @@ public class RoxygenCompletionTest {
             filtered.Should().BeEmpty();
         }
 
-        [Test]
-        public void Keywords02() {
+        [CompositeTest]
+        [InlineData("#  f", 1)]
+        [InlineData("#'  f", 2)]
```

Comment:
Do you care about construct `##`?

Reply:
It should be same as # (i.e. no Roxygen)